sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -715,6 +739,36 @@ func resourceSumologicMonitorsLibraryMonitorUpdate(d *schema.ResourceData, meta
 	if err != nil {
 		return err
 	}
+
+	// converting Resource FGP to Struct
+	permStmts, convErr := ResourceToCmfFgpPermStmts(d, monitor.ID)
+	if convErr != nil {
+		return convErr
+	}
+
+	// reading FGP from Backend to reconcile
+	fgpGetResponse, fgpGetErr := c.GetCmfFgp(fgpTargetType, monitor.ID)
+	if fgpGetErr != nil {
+		// if FGP endpoint is not enabled (not implemented) and FGP feature is not used,
+		// we should suppress this error
+		if len(HasErrorCode(fgpGetErr.Error(), []string{"not_implemented_yet"})) == 0 && len(permStmts) == 0 {
```

Comment:
`len(permStmts) == 0` should be done at L745. Why do we have it here?

Similar comment for error codes. We should add `api_not_enabled` code here too.

Reply:
@ayiusumo 
`len(permStmts) == 0 should be done at L745. Why do we have it here?` what do you suggest for this comment?